vendor/github.com/zclconf/go-cty-yaml/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/zclconf/go-cty-yaml
 
 require github.com/zclconf/go-cty v1.0.0
+
+go 1.13
```

Comment:
These `go` declarations don't seem to be present in the upstream `go.mod`s of these. Is this a new Go 1.13 or 1.14 vendoring behavior?

Reply:
That's interesting. I know that's a new behavior for go1.13, but I used go1.12 to create and test this PR. I assumed the go.mod files that showed up here were form their respective repos, so I'm not sure what's happening.

Reply:
I wonder if my editor jumped me into the cache directories and I ended updated these, since deleting and recreating everything in this repo gave the same result. Once I cleared out the cache, `go mod tidy` and `go mod vendor` fixed it.